src/components/DataBrowser.vue

```diff
@@ -197,7 +197,7 @@ export default {
         parentType: location._modelType,
         parentId: location._id,
         limit: itemsPerPage >= 0 ? itemsPerPage : null,
-        offset: (page - 1) * itemsPerPage,
+        offset: (page - 1) * Math.abs(itemsPerPage),
```

Comment:
Note: This handles when itemsPerPage is -1 (indicating no paging) and the result could become -0 which causes a problem in unit testing.

Reply:
`-0` _kills_ me.  Good catch.

Reply:
This is what happens when we only use IEEE 754 for all numeric types.